CHANGELOG.md

```diff
@@ -5,9 +5,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [1.0.0] - 2022-04-11
```

Comment:
This is currently curated manually, correct? If we keep up with using changesets, I wonder if a future enhancement could be to populate this automatically based on the changes since the last publish? Or at the very least, having individual changelogs in each of the sub-projects should make it easier to populate this manually without forgetting things.

Reply:
Yes, there is a way to automate the changelog with chagesets - there are some plugin type things we could use or roll our own, but I recall seeing one that will source changes via GitHub.

Something else we should discuss: Should we enforce changesets on all PRs?

Reply:
@CobyPear I think we should try enforcing changesets for PRs. If we don't, I worry that we won't keep up with them.